srcpkgs/calibre/template

```diff
@@ -19,14 +19,15 @@ depends="desktop-file-utils optipng poppler-utils python3-BeautifulSoup4
  python3-dnspython python3-feedparser python3-html5-parser python3-mechanize
  python3-msgpack python3-netifaces python3-psutil python3-regex python3-html2text
  python3-pychm qt5-webengine"
-checkdepends="${depends} libjpeg-turbo-tools jxrlib"
+checkdepends="${depends} libjpeg-turbo-tools jxrlib python3-pycryptodome
+ speech-dispatcher-devel"
```

Comment:
Are these libraries used exclusively for running tests? Otherwise it would make sense to add them to `depends`.

Reply:
Given https://github.com/archlinux/svntogit-community/blob/79e1a1f9018dd5efca5acccdbbae461d0d32692d/trunk/PKGBUILD#L68 , you can leave them in `checkdepends`, with a comment to check that `pycryptodome` is still required next time.

Reply:
ok, added comment

Reply:
pycryptodome is/was never used. It's apparently there "because some plugins might depend on it" but there's no stats on which or how many plugins those might be. This is likely more of an issue for standalone Windows/macOS builds where your thirdparty code cannot be fixed by `xbps-install python3-pycryptodome`. idk.

speech-dispatcher is intended, per upstream, to eventually be an optional dependency for text-to-speech in the ebook-viewer program. And I think it will only be used via the python API...

Reply:
Indeed, the test worked here with `speech-dispatcher`, the devel package isn't needed.

Thanks!